hcl/hcl_test.go

```diff
@@ -1097,6 +1145,9 @@ func testParser(t *testing.T, tc testcase) {
 
 // some helpers to easy build file ranges.
 func mkrange(start, end hhcl.Pos) hhcl.Range {
+	if start.Byte == end.Byte {
```

Comment:
Nice, I had some troubles with that but I missed the opportunity to improve this :heart: